indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -49,21 +60,46 @@ export default function EditableList() {
       codePresent={codePresent}
       editableType={type}
       eventId={eventId}
+      editors={editors}
     />
   );
 }
 
-function EditableListDisplay({editableList, codePresent, editableType, eventId}) {
+function EditableListDisplay({
+  editableList: origEditableList,
```

Comment:
Perhaps we could change this to `initialEditableList` to distinguish from a controlled prop, since we are immediately deriving state from it?

Reply:
good idea, initial* is also the name used to set the default value of other uncontrolled components